viewer/packages/360-images/src/Image360VisualizationBox.ts

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { SceneHandler } from '@reveal/utilities';
+import assert from 'assert';
+import { Image360Face } from '@reveal/data-providers';
+import { Image360Visualization } from './Image360Visualization';
+
+export class Image360VisualizationBox implements Image360Visualization {
+  private readonly _worldTransform: THREE.Matrix4;
+  private _visualizationMesh: THREE.Mesh | undefined;
+  private _faceMaterials: THREE.MeshBasicMaterial[] | undefined;
+  private readonly _sceneHandler: SceneHandler;
+
+  set opacity(alpha: number) {
+    if (this._faceMaterials === undefined) {
+      return;
```

Comment:
Should we keep track of opacity if it's set before materials are loaded?

(Same question goes for visibility, scale and render order, I guess)